content/geometry/object-vehicle/vehicle-structure.adoc

```diff
@@ -1374,11 +1485,14 @@ The indices are the same as in the parent group.
 |===
 
 
-== Grp_Exterior_Static_<part_idx>
+== Grp_Exterior_Static
+
+This group contains all static parts of the vehicle's exterior.
+Static elements are geometric structures that have a fixed position and
+orientation relative to the vehicle's origin throughout the simulation.
 
-This group contains all static elements of the exterior of the vehicle (part).
-Static elements are geometry structures that have a fixed position and orientation relative to the origin on the vehicle (part) throughout a simulation.
-They also do not change their state during the simulation, therefore lights are considered dynamic.
+In contrast to lights, which change their state depending on whether they are
```

Comment:
@ClemensLinnhoff: I rephrased this statement to enhance readability. Please check whether it is correct that lights are treated as dynamic elements because they can change state by switching on/off. Or is the main reason that the driver can adjust the direction of the light emission?

Reply:
Your phrasing is correct, thanks.